modules/@apostrophecms/widget-type/index.js

```diff
@@ -119,9 +119,15 @@ module.exports = {
     // two-thirds, half or full:
     width: '',
     // left or right:
-    origin: 'right'
+    origin: null,
```

Comment:
Why not using `auto` as default instead?
I think it will simplify the logic. The rest of the file will stay the same.

```suggestion
    // auto, left or right
    origin: 'auto',
```

Reply:
We can't do that. The goal here is to keep the `origin` default value to `right` so that we don't confuse everything that depends on that. While `auto` is possible, it will create a lot more problems and will greatly extend the scope, which is unnecessary. The `null` value is just internal logic.